core/backend/src/test/standalone/ViewStoreDb.test.ts

```diff
@@ -0,0 +1,332 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { expect } from "chai";
+import { Suite } from "mocha";
+import { join } from "path";
+import { Guid, GuidString, Logger, LogLevel, OpenMode } from "@itwin/core-bentley";
+import { ViewStore } from "../../ViewStore";
+import { ThumbnailFormatProps } from "@itwin/core-common";
+
+describe("ViewStore", function (this: Suite) {
+  this.timeout(0);
+
+  let vs1: ViewStore.ViewDb;
+
+  before(async () => {
+    Logger.setLevel("SQLite", LogLevel.None); // we're expecting errors
+    const dbName = join(__dirname, "output", "viewStore.db");
+    ViewStore.ViewDb.createNewDb(dbName);
+    vs1 = new ViewStore.ViewDb({ guidMap: {} as any });
+    vs1.openDb(dbName, OpenMode.ReadWrite);
+  });
+
+  after(async () => {
+    vs1.closeDb(true);
+    Logger.setLevel("SQLite", LogLevel.Error);
+  });
+
+  it("ViewDb", async () => {
+    const c1 = vs1.addCategorySelectorRow({ name: "first cat", json: "cat1-json" });
+    const ds1 = vs1.addDisplayStyleRow({ name: "first style", json: "style1-json" });
```

Comment:
Shouldn't we fail adding non-valid JSON strings?

Reply:
This test exists merely to validate that the SQL is working properly.